mmf/utils/modeling.py

```diff
@@ -15,7 +15,7 @@
 }
 
 
-def get_bert_configured_parameters(module, lr=None):
+def get_bert_configured_parameters(module, lr=None, weight_decay=0.01):
```

Comment:
We should also update the call sites to use `weight_decay` from `config.optimizer.params.get("weight_decay", 0.01)` in case users are using this.

Reply:
> We should also update the call sites to use weight_decay from config.optimizer.params.get("weight_decay", 0.01) in case users are using this.

I'm a bit not sure here. I feel changing call sites to `config.optimizer.params.get("weight_decay", 0.01)` would be a backward-incompatible change and could break some existing MMF models or user models.